server/api/product_handler.py

```diff
@@ -22,23 +20,31 @@ def oneProductRequests(product_id):
         db.session.delete(product)
         db.session.commit()
         return jsonify({'response': "Product '{}' was successfully deleted from the database".format(product_id)}), 200
-    
+
     if request.method == "PUT":
         req = request.get_json()
-        product = Product.query.get(int(product_id))
-        
-        if product:             
-            if "name" in req: product.name = req["name"]
-            if "old_price" in req: product.old_price = req["old_price"]
-            if "price" in req: product.price = req["price"]
-            if "url" in req: product.url = req["url"]
-            if "img_url" in req: product.img_url = req["img_url"]
+        try:
+            product = Product.query.get(int(product_id))
+            if "name" in req:
+                product.name = req["name"]
```

Comment:
We could make this more concise by using `product.name = req.get("name", product.name)`

Reply:
thanks carmen!